tests/roxctl/bats-tests/local/roxctl-netpol-generate-release.bats

```diff
@@ -3,6 +3,7 @@
 load "../helpers.bash"
```

Comment:
[not blocking the merge]
Is this file a copy-paste of `roxctl-netpol-generate-development.bats` with the only difference that `roxctl-development` is changed for `roxctl-release`? I don't think that massively copy-pasting anything is healthy practice that we can sustainably do. If there's no way to parameterize tests that they run development and release builds, I would vote to keep only the test that checks `roxctl-release`.

Reply:
Wouldn't it work to parameterize the specific parts with env vars and `source generic_test.sh` from two bats tests files? (didn't test)

Reply:
No, unfortunately sourcing would not work. There was a thread about this somewhere in the Bats GH/Wiki. Maybe I will give it a look if they got any new feats regarding this.